submissions/benjaminKamena/Dom/product.js

```diff
@@ -0,0 +1,64 @@
+// product validation
+let quantity = document.getElementById("quantity"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    total = document.getElementById("totalCost"),
+    submit = document.getElementById("btn"),
+    perUnit = price.innerHTML,
+    productPrice = perUnit * quantity.value;
+
+
+function quantityField(){
+    let productRegex = /^[1-9]+[0-9]*$/;
+    if(!quantity.value.match(productRegex)){
+        submit.disabled = true;
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+    } else{
+        submit.disabled = false;
+        quantity.nextElementSibling.innerHTML = "";
+    }
+}
+
+function priceField(){
+    if(quantity.value > 0){
+        productPrice = perUnit * quantity.value;
+        price.innerHTML = productPrice;
```

Comment:
I think you should call this `shippingAmount()` function here. Let this `shippingAmount()` takes productPrice as a parameter `shippingAmount(productPrice)`. Then pass it in here